pkg/network/utils.go

```diff
@@ -83,15 +83,17 @@ func DiscoverPTPDevices() ([]string, error) {
 			continue
 		}
 
-		// sysfs address looks like: /sys/devices/pci0000:17/0000:17:02.0/0000:19:00.5/net/eno1
+		var PCIAddr string
 		pathSegments := strings.Split(link, "/")
-		if len(pathSegments) != 8 {
+		if len(pathSegments) == 10 { // sysfs address for N3000 looks like: ../../devices/pci0000:85/0000:85:00.0/0000:86:00.0/0000:87:10.0/0000:8a:00.1/net/eth1
```

Comment:
Would taking the len() - 3 make this more general purpouse / more robust?

Also, if we pick the wrong segment the following if will fail and we will add the device to the list.

Would it make sense to validate the existance of /sys/bus/pci/devices/PCIAddr too ?